### PR TITLE
New How to

### PR DESCRIPTION
**Pull request by bmckercher123**
New How to "Determine best networking option" for Networking tab

- Can merge at any time
- Back ported to 3.10
- Casey and Matt reviewed

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 4d84db0d14c21ecbf7b2d5a69371ba9ad9e9e8ac

https://deploy-preview-2995--calico.netlify.com

File: _data/master/navbars/networking.yml [hunk 1]
```diff
@@ -4,26 +4,40 @@ order: 2
 toc:
 - title: Networking
   path: /networking/
+
+- title: Determine best networking option
+  path: /networking/determine-best-networking
+  
 - title: Connecting workloads across networks that you do not control
   path: /networking/vxlan-ipip
+
```
Comment (caseydavenport) [inline _data/master/navbars/networking.yml]: Could you undo these newline changes?

File: master/networking/determine-best-networking.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+This option also provides near host-to-host levels of performance and allows the network direct visibility of traffic.
+
+{{site.prodname}} can route pod traffic between nodes without encapsulation when all nodes are on a single L2 subnet, and if the underlying network doesn’t enforce IP address checks.  If your network consists of multiple L2 subnets then you can either [peer over BGP with your routers]({{site.baseurl}}/{{page.version}}/networking/bgp), or use [cross-subnet encapsulation](#ip-in-ip-or-vxlan-encapsulation) to encapsulate only traffic that crosses subnet boundaries.
+  
+Traffic cannot be routed between pods and destinations that aren’t also in the {{site.prodname}} cluster without allowing workload access outside cluster, or peering with infrastructure.
+
+> **Tip**: On AWS, you can disable source/destination checking to use this option within a VPC subnet. [Can I run Calico in a public cloud environment?]({{site.baseurl}}/{{page.version}}/reference/faq#can-i-run-calico-in-a-public-cloud-environment).
+{: .alert .alert-info}
+
+#### IP in IP or VXLAN encapsulation
```
Comment (caseydavenport) [inline master/networking/determine-best-networking.md]: We should probably link to this document somewhere in this section: https://docs.projectcalico.org/v3.10/networking/vxlan-ipip

File: _data/master/navbars/networking.yml (hunk 1 above)
Comment (bmckercher123) [reply _data/master/navbars/networking.yml]: fixed.

File: master/networking/determine-best-networking.md (hunk 2 above)
Comment (bmckercher123) [reply master/networking/determine-best-networking.md]: Added.

File: _data/v3_10/navbars/networking.yml [hunk 3]
```diff
@@ -4,26 +4,49 @@ order: 2
...
+
+- title: Determine best networking option
+  path: /networking/determine-best-networking
+  
 - title: Connecting workloads across networks that you do not control
   path: /networking/vxlan-ipip
+
 - title: Migrate from one IP pool to another
   path: /networking/migrate-pools
+
```
Comment (caseydavenport) [inline _data/v3_10/navbars/networking.yml]: Could you do these lines too?

Comment (bmckercher123) [reply _data/v3_10/navbars/networking.yml]: fixed.

**Pull request merged**